tests/bud/context-escape-dir/upperfile.txt

```diff
@@ -0,0 +1,3 @@
+This is a text file to be used in Buildah testing.
+This will be used to ensure that a file from above the context 
+directory can not be copied during the buildh phase.
```

Comment:
apparently a rogue space at the end of this line caused tests to be unhappy.  Also touched up the "buildh" typo